coalib/misc/CachingUtilities.py

```diff
@@ -84,7 +84,7 @@ def pickle_load(log_printer, identifier, fallback=None):
                         Otherwise the ``default`` value is returned.
     """
```

Comment:
And the commit body says `Null` when it should say `None`. Sorry for the nitpicking :)